src/app/components/TransactionsTable/index.tsx

```diff
@@ -60,9 +64,19 @@ export default function TransactionsTable({
                 <div className="flex gap-3">
                   <div className="flex items-center">
                     {type == "outgoing" ? (
-                      <div className="flex justify-center items-center bg-orange-100 dark:bg-orange-950 rounded-full w-8 h-8">
-                        <PopiconsArrowUpSolid className="w-5 h-5 text-orange-400 dark:text-amber-600 stroke-[1px] stroke-orange-400 dark:stroke-amber-600" />
-                      </div>
+                      tx.state === "pending" ? (
+                        <div className="flex justify-center items-center bg-orange-100 dark:bg-orange-950 rounded-full w-8 h-8">
```

Comment:
Can you check the styles in Alby Go? it uses a pulse animation rather than orange

Reply:
added animated pulse animation. for outgoin transactioons we use bg-orange. so i am using orange bg for sent as well as sending. sending have pulse animation now by default

for failed i changed things to red